### PR TITLE
Add cargo shuttle

### PR DESCRIPTION
**Pull request by metalgearsloth**
Need to detach the orders console from telepad / shuttle first then shuttle comes after. We already settled on cargo pads being the sell points on it so no need to teardown / setup shuttle.

As part of this the cargo DB was moved to the station and no more machine linking is needed between the orders console and the telepad etc.

Fixes https://github.com/space-wizards/space-station-14/issues/554 and https://github.com/space-wizards/space-station-14/issues/3762 and https://github.com/space-wizards/space-station-14/issues/5563

CargoPilotConsole is pretty crappy but for remote piloting I'd prefer to have an actual UI and interface with device networks. This PR is already pretty hefty even without the additional map so I left it out.

I want to move the recall button onto the shuttle console instead, for now I just usually guess 200m is far enough to recall, but I will move it with the evac shuttle PR and turn it into a hyperspace destination.

Prototype renames:
- cargoTelepad -> CargoTelepad
- ComputerSupplyRequest -> ComputerCargoOrders

:cl: metalgearsloth & Emisse
- add: A cargo shuttle has been added.
- tweak: Cargo requests and orders consoles have been combined. Either can be used to request items but only cargo techs can approve still.
- tweak: Cargo orders now prefill your name. The window has also been made wider to compensate.
- tweak: Cargo bank passive income has been reduced from 15 -> 5 per second.
- tweak: Telepad time increased from 30 -> 45 seconds, but it will automatically grab station orders.

### CODE REVIEW

File: Content.Server/Cargo/Systems/CargoSystem.Console.cs [hunk 1]
```diff
@@ -24,12 +24,12 @@ public sealed partial class CargoSystem
...
         /// </summary>
-        private const float Delay = 10f;
+        private const int Delay = 10;
 
         /// <summary>
-        /// How many points to give to every bank account every <see cref="Delay"/> seconds.
+        /// How many points to give to every bank account every second.
         /// </summary>
-        private const int PointIncrease = 10000;
+        private const int PointIncrease = 5;
```
Comment (moonheart08) [inline Content.Server/Cargo/Systems/CargoSystem.Console.cs]: Shouldn't these be part of the stationbankaccountcomponent? That'd also allow them to be configured per-station in StationConfig/StationInfo (I forgot what I named it and cba to check).

Comment (metalgearsloth) [reply Content.Server/Cargo/Systems/CargoSystem.Console.cs]: I did I just haven't pushed it.

File: Content.Client/Content.Client.csproj [hunk 2]
```diff
@@ -23,6 +23,11 @@
     <ProjectReference Include="..\RobustToolbox\Robust.Client\Robust.Client.csproj" />
     <ProjectReference Include="..\Content.Shared\Content.Shared.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Cargo\UI\CargoConsoleMenu.xaml.cs">
+      <DependentUpon>CargoConsoleMenu.xaml</DependentUpon>
+    </Compile>
+  </ItemGroup>
```
Comment (moonheart08) [inline Content.Client/Content.Client.csproj:30]: ?

File: Content.Shared/CCVar/CCVars.cs [hunk 3]
```diff
@@ -160,7 +160,7 @@ public sealed class CCVars : CVars
         ///     Controls if the game should obey map criteria or not. Overriden if a map vote or similar occurs.
         /// </summary>
         public static readonly CVarDef<bool>
-            GameMapForced = CVarDef.Create("game.mapforced", false, CVar.SERVERONLY);
+            GameMapForced = CVarDef.Create("game.mapforced", true, CVar.SERVERONLY);
```
Comment (moonheart08) [inline Content.Shared/CCVar/CCVars.cs:163]:
```suggestion
            GameMapForced = CVarDef.Create("game.mapforced", false, CVar.SERVERONLY);
```

**Review by moonheart08 — CHANGES_REQUESTED** [review]

**Review by mirrorcult — CHANGES_REQUESTED** [review]
- seems like there are some mapping issues. saltern for example has two ordering computers but not a shuttle computer, looks like the telepads got replaced with an order computer and not a shuttle computer. none of them have the pilot consoles either
- the cargo shuttle computer only opens the UI when you click on it twice? after you open it for the first time

File: Content.Client/Content.Client.csproj (hunk 2 above)
Comment (metalgearsloth) [reply Content.Client/Content.Client.csproj:30]: rider moment

File: Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+        }
+
+        // It gets mapinit which is okay... buuutt we still want it paused to avoid power draining.
+        CargoMap = _mapManager.CreateMap();
+        _mapManager.SetMapPaused(CargoMap!.Value, true);
+
+        foreach (var comp in EntityQuery<StationCargoOrderDatabaseComponent>(true))
+        {
+            AddShuttle(comp);
+        }
```
[the remark this answers is not in the document]
Comment (metalgearsloth) [reply Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs:539]: I already subbed but I did it so it's created at roundstart. I just changed it so Setup is called during the station startup.

**Pull request merged**